app/src/main/java/com/quarkworks/apartmentgroceries/auth/SignUpActivity.java

```diff
@@ -37,15 +41,18 @@ protected void onCreate(Bundle savedInstanceState) {
 
         usernameEditText = (EditText) findViewById(R.id.sign_up_username_id);
         passwordEditText = (EditText) findViewById(R.id.sign_up_password_id);
-        secondPasswordEditText = (EditText) findViewById(R.id.sign_up_second_password_id);
+        confirmPasswordEditText = (EditText) findViewById(R.id.sign_up_confirm_password_id);
         signUpButton = (Button) findViewById(R.id.sign_up_submit_button_id);
+        toolbar = (Toolbar) findViewById(R.id.main_toolbar_id);
+        titleTextView = (TextView) toolbar.findViewById(R.id.toolbar_title_id);
```

Comment:
It's a good organization practice to keep the code where you get view references separate from the code where you set view data.